junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/execution/ConditionEvaluator.java

```diff
@@ -113,7 +114,7 @@ private String getDeactivatePatternString(ConfigurationParameters configurationP
 	}
 
 	/**
-	 * See {@link Constants#DEACTIVATE_CONDITIONS_PATTERN_PROPERTY_NAME} for
+	 * See {@link org.junit.jupiter.engine.Constants#DEACTIVATE_CONDITIONS_PATTERN_PROPERTY_NAME} for
```

Comment:
This can now point to the constant defined above instead.

Reply:
Didn't copy the Javadoc over to the local constant on purpose. So linking to the "published" constant with its Javadoc seems better to me.

Reply:
Fair enough.